monitoring/rid_qualifier/test_executor.py

```diff
@@ -0,0 +1,34 @@
+from monitoring.rid_qualifier.aircraft_state_replayer import TestHarness, TestBuilder
+import asyncio
+import arrow
+from monitoring.rid_qualifier.utils import RIDQualifierTestConfiguration, RIDQualifierUSSConfig
+
+def build_uss_config(injection_base_url:str, allocated_track:int = 0) -> RIDQualifierUSSConfig:
+  return RIDQualifierUSSConfig(injection_base_url=injection_base_url, allocated_flight_track_number = allocated_track)
+  
+
+def build_test_configuration(locale: str, auth_spec:str, uss_config:RIDQualifierUSSConfig) -> RIDQualifierTestConfiguration: 
+    now = arrow.now()
+    test_start_time = now.shift(minutes=3) # Start the test three minutes from the time the test_exceutor is run. 
+    
+    test_config = RIDQualifierTestConfiguration(
+      locale = locale,
+      now = now.isoformat(),
+      test_start_time = test_start_time.isoformat(),
+      auth_spec = auth_spec,
+      usses = [uss_config]
+    )
+
+    return test_config
+    
+async def main(test_configuration: RIDQualifierTestConfiguration):
```

Comment:
Does this method need to be async?

Reply:
Same as above, I am removing all async references / import and code